iio/iiod.h

```diff
@@ -48,6 +48,11 @@
 #define IIOD_VERSION		"1.1.0000000"
 #define IIOD_VERSION_LEN	(sizeof(IIOD_VERSION) - 1)
 
+#define MAX_DEV_ID		21
```

Comment:
I'd say that rather do some bug hunting in the future, perhaps it's better to just increase 21 and 22 to 32 or 64. It's still better than 256 (linux) but large enough so that we won't ever be increasing it again.

Reply:
done. thanks!